recipes/apsw/meta.yaml

```diff
@@ -15,14 +15,14 @@ source:
 build:
   number: 0
   script:
-    - python setup.py fetch --sqlite --version {{ sqlite_version }}  
     - python setup.py install --single-version-externally-managed --record record.txt
     - python setup.py test
   
 requirements:
   build:
     - python
     - setuptools
+    - sqlite {{ sqlite_version }}
```

Comment:
Please don't pin with a jinja variable b/c that will probably break the conda-forge's pinning bot.